homeassistant/components/yamaha_musiccast/const.py

```diff
@@ -51,10 +47,10 @@
 }
 
 ENTITY_CATEGORY_MAPPING = {
-    EntityType.CONFIG: ENTITY_CATEGORY_CONFIG,
+    EntityType.CONFIG: EntityCategory.CONFIG,
     EntityType.REGULAR: None,
-    EntityType.DIAGNOSTIC: ENTITY_CATEGORY_DIAGNOSTIC,
-    EntityType.SYSTEM: ENTITY_CATEGORY_SYSTEM,
+    EntityType.DIAGNOSTIC: EntityCategory.DIAGNOSTIC,
+    EntityType.SYSTEM: EntityCategory.SYSTEM,
```

Comment:
This entity category should not be used in general, what is it used for?

Reply:
In aiomusiccast small dedicated features such as the dialogue volume, equalizer and others are implemented as capabilities. These capabilities are used to expose additional select and number entities in Home Assistant. As the capabilities have a type, which is defined in aiomusiccast, we use this mapping to map aiomusiccast EntityTypes to Home Assistant EntityCategories. 
I believe all currently implemented aiomusiccast capabilities, which are used in Home Assistant, are config entities, but we want to add some diagnostic sensors for video resolutions and audio codecs in the near future. Why shouldn't the categories Diagnostic and System be used in general?

Reply:
I know hoe entity categories work... it is specifically the system entity category (which is the line this comment was left on).

In general, the system category should not be used, however, it is here; hence me asking about the use of the system category in this integration.

Reply:
Mapping it to Diagnostic is fine, I guess. As long as they are read-only (which is what should be classified as diagnostic), else it would be config if they are read-write.

IMHO, we should remove it to prevent confusion or unexpected use.

Reply:
Right, I just checked aiomusiccast and as we are on the latest release anyway, I just created a new minor release and removed the EntityType.SYSTEM there so that we can safely drop it in HA, too